### ISSUE TITLE
Unable to generate a accessibility report for a pop up message on web browser

### ISSUE THREAD

**Issue by Shaheerlatif1** [GoogleChrome/lighthouse#13890]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://public-portal-qa.accertaweb.local/

### What happened?

There is a sign up and register page I clicked register a pop up is displayed with 3 option and some description. I need to check the accessibility for that pop up. When I generate the accessibility report the page got refresh and the pop up is gone. 

### What did you expect?

Accessibility report should be generated for a pop up window 

### What have you tried?

There is a sign up and register page I clicked register a pop up is displayed with 3 option and some description. I need to check the accessibility for that pop up. When I generate the accessibility report the page got refresh and the pop up is gone. 

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

9.4.0

### Chrome Version

100.0.4896.127

### Node Version

.

### OS

Windows

### Relevant log output

```shell
There is a sign up and register page I clicked register a pop up is displayed with 3 option and some description. I need to check the accessibility for that pop up. When I generate the accessibility report the page got refresh and the pop up is gone.
```
